### ISSUE TITLE
tModLoader can't detect my headphones

### ISSUE THREAD

**Issue by JoshPKT** [bluemagic123/tModLoader#2949]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

Since updating to the most recent TML version, whenever I start it up I get no audio coming in through my headphones. Furthermore, unplugging the headphone cable while TML is running causes an instant crash. I've done pretty extensive troubleshooting and concluded that the issue definitely lies within this version of TML, not my headphones or my system or Terraria itself or anything like that. Interestingly, if I switch my headphones to wireless it seems to resolve the issue (though this isn't a good solution for me since there's about a half second delay when using my headphones wirelessly.)

### Log File

[client.log](https://github.com/tModLoader/tModLoader/files/9601551/client.log)

### Steps to reproduce

1. Plug in headphones (Bose Quietcomfort 35 ii) via heaphone jack.
2. Startup TML
3. Unplug headphone cable

### Expected Behavior

After Step 2, the program should automatically detect the headphones as the audio device through which to play audio.
After Step 3, the program should revert to the default audio device (i.e. desktop audio).

### Actual Behavior

After Step 2, the program does now play audio through the headphones.
After Step 3, the program crashes.

### Reproduction frequency

On my end, this is 100% consistent behavior.

**Comment by Chicken-Bones** [comment]
Duplicate of #2797, Please follow that issue for updates. I'll message here or try and find you on Discord if there's any way I can figure out to test/fix this

**Comment by JoshPKT** [comment]
Oh ok thanks for letting me know. My discord is @dialog_box#3936. I also
made a bug report in the TML discord in #support-forums.
